packages/benchmark/src/n8nApiClient/n8nApiClient.types.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * n8n workflow. This is a simplified version of the actual workflow object.
+ */
+export interface Workflow {
```

Comment:
Import and use `Omit`?

Reply:
I added this but had to revert as it unncessarily complicated the Docker build (would need to copy and build the workflow package as well). Didn't feel worth it to just get a single type from the package